### DISCUSSION TITLE
Gateway timeout when deleting many (?) objects or a bigger (?) object

### DISCUSSION THREAD

**Discussion by docwarems** [minio/minio#16384] [Q&A] [answered]
Hi, I'm unsure whether I should create an issue, so I copied the issue template and filled it, to first dicuss here.

## Expected Behavior
I try to to delete a folder containing 78 objects, each of 1 MB. Web console oder mc should be able to delete folder.
or
select bigger object (35 MB) and delete it, Web console oder mc should be able to delete file.

## Current Behavior
 After several minutes a "Gateway timeout 504" occurs, both with web console and mc. Folder not deleted. Folder still contains 78 objects.
or 
Bigger file still there

## Possible Solution
- Filter objects with web console to reduce object number
- select these objects
- delete -> sucessful

## Steps to Reproduce (for bugs)
This occurs on a private cloud

## Context
I'm doing scaling tests. After each test I want to delete the generated MinIO objects. 

## Regression
I haven't done bulk deletes so far, so I cannot say it it's regression issue.

## Your Environment

" mc admin info <alias>" prints version 2022-08-02T23:59:16Z
OS: linux
I'm only minIO User so I cannot give more info currently. But I could forward more specific questions to our admins.

**Comment by klauspost** [comment, marked as answer]
There shouldn't really be any gateway timeouts set, though any reasonable hardware should be able to complete that operation very fast.

Without any more information on the actual setup, it is hard to help more.

Sounds like a very broken setup.

**Comment by docwarems** [comment]
Yes, there must be some server issue we have to look at. Today I could delete all the folders, I yesterday got these 504 errors for.
